Networking/Networking/Mapper/MetaDataMapper.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+/// Mapper: MetaData List
+///
+struct MetaDataMapper: Mapper {
+
+    /// (Attempts) to convert an instance of Data into an array of MetaData Entities.
+    ///
+    func map(response: Data) throws -> [MetaData] {
+        let decoder = JSONDecoder()
+        if hasDataEnvelope(in: response) {
+            let envelope = try decoder.decode(DataEnvelope.self, from: response)
+            return envelope.data.metadata.filter { !$0.key.hasPrefix("_") }
+        } else {
+            let envelope = try decoder.decode(MetaDataEnvelope.self, from: response)
+            return envelope.metadata.filter { !$0.key.hasPrefix("_") }
```

Comment:
Can we add a comment to explain why we filter these values with `_` prefix?

Reply:
Good suggestion, done 3fba503a2358613a8031ab9322c3c9d2c5e1d7cb